data/json/npcs/missiondef_robofac.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "id": "MISSION_ROBOFAC_INTERCOM_1",
+    "type": "mission_definition",
+    "name": "Return Field Data",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 5,
+    "item": "robofac_test_data",
+    "value": 1000,
+    "start": {
+      "assign_mission_target": { "om_terrain": "field", "reveal_radius": 2, "random": true, "search_range": 10 },
+      "update_mapgen": {
+        "rows": [
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "                        ",
+          "          M             ",
+          "            b b         ",
+          "         b b b@bb       ",
+          "             b b        ",
+          "                        ",
+          "                        "
+        ],
+        "terrain": { "@": "t_grass", "b": "t_dirt", "M": "t_dirt" },
+        "fields": { "b": { "field": "fd_blood", "density": 1 } },
+        "mapping": { "@": { "items": { "item": "produce" }, "item": [ { "item": "EMPbomb", "count": 2 }, { "item": "corpse" } ] } },
+        "monster": { "M": { "monster": "mon_robofac_prototype" } }
+      }
+    },
+    "end": {
+      "effect": [
+        { "npc_add_var": "npc_intercom_trade", "type": "dialogue", "context": "intercom", "value": "yes" },
+        { "u_add_var": "completed_robofac_intercom_1", "type": "dialogue", "context": "intercom", "value": "yes" }
+      ]
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "followup": "MISSION_FREE_MERCHANTS_EVAC_3",
+    "dialogue": {
+      "describe": "No I said ... [*You Hear a short, muffled conversation from across the intercom*] /n Well it seems we do have a use for you.  It's dangerous and you are likely to die, but if you complete it we will allow you limited access to our resources.",
```

Comment:
This line is actually never said, and I'm unsure why.

On the same line, I was wodering if its possible to override the TALK_MISSION_LIST Npc dialogue, I see its appears  partly jsonized, but I dont understand how the replies connect with each other.